### PR TITLE
Element インターフェイスの記事を更新

### PR DESCRIPTION
**Pull request by mfuji09**

### CODE REVIEW

**Comment by sergiogval** [comment]
May I ask for a good reference to get full insight about macros? I am kind of noob using them. And it seems to me it will be necessary to have a solid foundation.

**Comment by mfuji09** [comment]
@sergiogval If you are looking for references of macros, you can find them in https://developer.mozilla.org/en-US/docs/MDN/Structures/Macros .

**Review by potappo — CHANGES_REQUESTED** [review]
脱字を修正しましたので、確認をお願いします。

File: files/ja/web/api/element/index.md [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+
+例えば {{DOMxRef("HTMLElement")}} インターフェイスは HTML 要素の基本インターフェイスであり、 {{DOMxRef("SVGElement")}} インターフェイスはすべての SVG 要素の基本になります。ほとんどの機能は、クラスの階層を下りると具体化していきます。
+
+XUL の `XULElement` インターフェイスのようなウェブプラットフォームの枠外の言語もまた、`Element` インターフェイスを実装しています。
+
+{{InheritanceDiagram}}
+
+## プロパティ
+
+_`Element` は、親インターフェスである {{DOMxRef("Node")}}、およびその親インターフェイスである {{DOMxRef("EventTarget")}} からプロパティを継承しています。_
```
[the remark this answers is not in the document]
Comment (mfuji09) [reply files/ja/web/api/element/index.md]: ありがとうございます！

**Pull request merged**